frontend/packages/knative-plugin/src/components/overview/__tests__/ConfigurationsOverviewListItem.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ConfigurationModel } from '@console/knative-plugin';
+import ConfigurationsOverviewListItem from '../ConfigurationsOverviewListItem';
+import { mockConfigurationsData } from '../__mocks__/overview-knative-mock';
+
+describe('ConfigurationsOverviewListItem', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should list the Configuration', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.find('li')).toHaveLength(1);
```

Comment:
I've changed my mind and I think the root element being an `li` is important to the API but not just finding at any location. We should assert the root.
`expect(wrapper.type()).toBe('li');`

Reply:
@nemesis09 what about this one? You're still doing a `find` which could be anywhere in the markup instead of just the root.

Reply:
missed that one. Fixed it.